packages/contracts/test/multisig/asset_proxy_owner.ts

```diff
@@ -422,6 +430,33 @@ describe('AssetProxyOwner', () => {
                 expect(isAuthorized).to.equal(false);
             });
 
+            it('should execute removeAuthorizedAddressAtIndex for registered address if fully confirmed and called by non-owner', async () => {
+                const removeAuthorizedAddressAtIndexData = erc20Proxy.removeAuthorizedAddressAtIndex.getABIEncodedTransactionData(
+                    authorized,
+                    erc20Index,
+                );
+                const submitRes = await multiSigWrapper.submitTransactionAsync(
```

Comment:
Nit - add sanity check that `authorized` is an authorized address before removing it.